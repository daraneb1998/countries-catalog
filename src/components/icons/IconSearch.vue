<template>
  <svg
    aria-hidden="true"
    class="w-5 h-5"
    fill="none"
    stroke="currentColor"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
    data-v-65e472f8=""
  >
    <path
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
      d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
      data-v-65e472f8=""
    ></path>
  </svg>
</template>
