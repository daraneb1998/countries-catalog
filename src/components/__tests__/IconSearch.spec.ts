import { describe, it, expect } from "vitest";

import { mount } from "@vue/test-utils";
import IconSearch from "../icons/IconSearch.vue";

describe("IconSearch", () => {
  it("renders properly", () => {
    const wrapper = mount(IconSearch);
    expect(wrapper.text()).toMatchSnapshot();
  });
});
