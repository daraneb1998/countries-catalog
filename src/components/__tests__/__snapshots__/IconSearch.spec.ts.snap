// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IconSearch > renders properly 1`] = `""`;
